spawn_worker/src/unix/spawn.rs

```diff
@@ -8,7 +8,7 @@ mod linux {
     use std::io::{Seek, Write};
 
     pub(crate) fn write_memfd(name: &str, contents: &[u8]) -> anyhow::Result<memfd::Memfd> {
-        let opts = memfd::MemfdOptions::default();
+        let opts = memfd::MemfdOptions::default().close_on_exec(false);
```

Comment:
Why not add the comment that @bwoebi made here, so it's clear that this is critical and ok to _leak_ the `fd`.